### ISSUE TITLE
I'd like to don't show all data values on radar tooltip on radar chart, only the indicator one

### ISSUE THREAD

**Issue by daniellycosta** [apache/incubator-echarts#11323]
### What problem does this feature solve?
I have a  radar chart with 21 indicators and is very confuse see all the data values on mouse hover as you can see on the link below

[Radar Chart - Example 1](https://codepen.io/daniellycosta/pen/XWrGVXw)

I'd like to have something like this  [Radar Chart - Example 2](https://codepen.io/daniellycosta/pen/eYOXyzo). The tooltip only showing the value of the item the user puts the mouse on

### What does the proposed API look like?
If the `dataIndex` param show the index of the item It would be possible to use the second example of formatting function

#### Params Json
```
{
"componentType":"series",
"componentSubType":"radar",
"componentIndex":0,
"seriesType":"radar",
"seriesIndex":0,
"seriesId":"\u0000预算 vs 开销（Budget vs spending）\u00000","seriesName":"预算 vs 开销（Budget vs spending）",
"name":"",
"dataIndex":0,
"data":[43,100,28,35,50,19,43,100,28,35,50,43,100,28,35,50,43,100,28,35,50,89],
"value":[43,100,28,35,50,19,43,100,28,35,50,43,100,28,35,50,43,100,28,35,50,89],
"color":"#c23531",
"marker":"<span style=\"display:inline-block;margin-right:5px;border-radius:10px;width:10px;height:10px;background-color:#c23531;\"></span>",
"$vars":["seriesName","name","value"]
}
```

#### Formatting function I have used on the second Example

```
formatter: function(params) {
    return params.value[params.dataIndex];
 }
```

**Comment by lahvey** [comment]
yes, tooltip and radar axisLabel have the same formatter problem, expect to fix them both.

**Comment by BEGEMOT9I** [comment]
@pissang Hi. When it will be fixed? Now we haven't got an ability to get info about value or index of value in radar chart (i don't speak only about tooltip, but about handlers too). Could you implement a `valueIndex` for event and tooltip?
